app/models/container_node.rb

```diff
@@ -115,4 +114,12 @@ def external_logging_path
     index = ".operations.*"
     EXTERNAL_LOGGING_PATH % {:index => index, :query => query}
   end
+
+  def disconnect_inv
+    return if ems_id.nil?
```

Comment:
Yeah, sounds like a followup PR to clean them all up.

Might be the ```ems_id.nil?``` is still relevant when we destroy EMS? At least in cloud/infra, we do dependent => nullify for some models. (to speed up the destroy)